packages/blade/src/tokens/theme/bankingTheme.ts

```diff
@@ -1067,12 +1067,12 @@ const colors: ColorsWithModes = {
       },
       border: {
         normal: {
-          lowContrast: globalColors.neutral.navyGrayDark[600],
-          highContrast: globalColors.neutral.navyGrayDark[600],
+          lowContrast: globalColors.neutral.navyGrayDark.a100,
```

Comment:
@saurav12 this was already merged right? why is this change coming again? is this new update? cc @chaitanyadeorukhkar

Reply:
@kamleshchandnani No no, I've changed it yesterday only.

Reply:
Looks like another change

Reply:
Wait! Do not merge this

Reply:
All good, this can be merged!